scripts/analysisTools/w_mass_13TeV/run2Dsmoothing.py

```diff
@@ -774,16 +805,35 @@ def runSmoothing(inputfile, histname, outdir, step, args, effHist=None):
     )
     args = parser.parse_args()
 
-    if args.isolationDefinition == "iso04":
-        effSmoothFile = sfFolder + "efficiencies3D_rebinUt2.pkl.lz4"
-        #
-        inputRootFile = {
-            "iso": f"{sfFolder}iso3DSFVQTextended.root",
-            "isonotrig": f"{sfFolder}isonotrigger3DSFVQTextended.root",
-            "isoantitrig": f"{sfFolder}isofailtrigger3DSFVQTextended.root",
-            "triggerplus": f"{sfFolder}triggerplus3DSFVQTextended.root",
-            "triggerminus": f"{sfFolder}triggerminus3DSFVQTextended.root",
-        }
+    if not hasattr(args, "indir"):
+        args.indir =  data_dir + "/muonSF/"
+        logger.warning("Input folder for 3d efficiencies not specified. Using the default path.")
+
+
+    if args.era == "2016PreVFP":
+        logger.error("Run 2016 PreVFP is not currently available!")
```

Comment:
Probably better just not to include preVFP in the list of possible choices in the options, rather than allowing it and then throwing here.